src/Notes/WC_Admin_Notes.php

```diff
@@ -19,11 +19,18 @@ class WC_Admin_Notes {
 	 */
 	const UNSNOOZE_HOOK = 'wc_admin_unsnooze_admin_notes';
 
+	/**
+	 * Hook used for recurring "action" action.
```

Comment:
What do you think about using "expiration" or "expire" instead of "action" as a verb? (I know we're already doing that with status, but it's a little awkward in my opinion)

Reply:
I see both sides to this one myself. `action` is definitely kind of a loaded word in WP-nomenclature. But also it is the name of how we track this in the DB. My other thought here is should we just add the call of `::action_notes()`  into `[https://github.com/woocommerce/woocommerce-admin/blob/master/src/Events.php#L64](do_wc_admin_daily)` - that way we aren't adding in another recurring job?

Reply:
I would prefer using an existing hook as @timmyc suggests, but is once a day going to be enough for this? It could potentially sit there for 23 hours with an expired nonce.